ext/systemd/pe-bolt-server.service

```diff
@@ -3,7 +3,8 @@ Description=PE Bolt Server
 After=syslog.target network.target
 
 [Service]
-Type=simple
+Type=fork
```

Comment:
The init service script expects a daemonized service that writes a pidfile. Updated systemd to expect the same thing for consistency.